lib/Behavior/Mapping/LocaleBehavior.php

```diff
@@ -29,4 +29,18 @@ public function getLocale();
      * @param string $locale
      */
     public function setLocale($locale);
+
+    /**
+     * Return the documents original locale.
+     *
+     * @return string
+     */
+    public function getOriginalLocale();
+
+    /**
+     * Sets the document original locale.
+     *
+     * @param string $locale
+     */
+    public function setOriginalLocale($locale);
```

Comment:
I am quite skeptical about this change... The original locale makes only sense for documents with the `ShadowLocaleBehavior`, doesn't it? This behavior already has a `shadowLocale`, which would be duplicating information then.